### ISSUE TITLE
jsdoc

### ISSUE THREAD

**Issue by germsb** [johnsoncodehk/volar#105]
First of all thank you for the work.

Is it possible to support the documentation ? e.g :
```ts
const props = defineProps({
    /** A beautiful props */
    animatePopup: {
        default: true,
        type: Boolean,
    }
})
```
![Imgur](https://i.imgur.com/5pajOVA.jpg)

**Comment by johnsoncodehk** [comment]
Thanks for the suggestion! I will take a look.

**Comment by johnsoncodehk** [comment]
Supported in 0.22.18.

**Comment by germsb** [comment]
Thank you 

Maybe a side effect, I get an error when hovering the slotProp declaration like this:
` <slot name="default" :showPopup="showPopup" :hidePopup="hidePopup">`
```js
//...
/** bla bla */
function showPopup() { /*stuff*/}
```
```console
[Error - 18:40:33] Request textDocument/hover failed.
  Message: Request textDocument/hover failed with message: Maximum call stack size exceeded
  Code: -32603 
```

**Comment by johnsoncodehk** [comment]
@germsb Fixed in 0.22.19, thank you.
